jerry-core/ecma/operations/ecma-function-object.c

```diff
@@ -1603,6 +1608,11 @@ ecma_op_external_function_list_lazy_property_names (uint32_t opts, /**< listing
 {
   JERRY_UNUSED (main_collection_p);
 
+  if (opts & ECMA_LIST_ARRAY_INDICES)
```

Comment:
Could we handle these in the caller, and not call the function in the first place?